README.md

```diff
@@ -1,17 +1,43 @@
+
 <img src="https://user-images.githubusercontent.com/39308480/43605302-e794780c-9665-11e8-9e25-7abefc7a3092.png" alt="jelly-beats" width="54%">
 
-# Decentralized music streaming platform
+# Decentralized music streaming platform [![David](https://img.shields.io/david/btzr-io/jelly-beats.svg?style=flat-square)](https://david-dm.org/btzr-io/jelly-beats)[![GitHub license](https://img.shields.io/github/license/btzr-io/jelly-beats.svg?style=flat-square)](https://github.com/btzr-io/electron-preact-app/blob/master/LICENSE)
 
-[![David](https://img.shields.io/david/btzr-io/jelly-beats.svg?style=flat-square)](https://david-dm.org/btzr-io/jelly-beats)
-[![GitHub license](https://img.shields.io/github/license/btzr-io/jelly-beats.svg?style=flat-square)](https://github.com/btzr-io/electron-preact-app/blob/master/LICENSE)
+> What? A decentralized music streaming platform.
+How? We are developing an open source desktop application using the electron framework, react, web workers, unistore.js and the lbry-protocol.
+**Disclaimer:**
+We don’t intend to develop a real working product that can compete or replace other centralized platforms such as soundcloud, spotify, deezer (At least not right now).
+Our main goal right now it’s to develop a simple but functional application to showcase the current potential of the lbry-protocol for media distribution and discovery using simple web technologies.
```

Comment:
Did you just copy paste this from the fund proposal ? :stuck_out_tongue:

Reply:
I don't think we need this ^^

Reply:
I was thinking its very cool :D

Reply:
@btzr-io Should be ok now :)